src/components/recordingcreator/seriesrecordingeditor.js

```diff
@@ -1,143 +1,200 @@
+function init(context) {
+    fillKeepUpTo(context);
 
-        apiClient.getLiveTvSeriesTimer(currentItemId).then(function (item) {
-            item.PrePaddingSeconds = form.querySelector('#txtPrePaddingMinutes').value * 60;
-            item.PostPaddingSeconds = form.querySelector('#txtPostPaddingMinutes').value * 60;
-            item.RecordAnyChannel = form.querySelector('.selectChannels').value === 'all';
-            item.RecordAnyTime = form.querySelector('.selectAirTime').value === 'any';
-            item.RecordNewOnly = form.querySelector('.selectShowType').value === 'new';
-            item.SkipEpisodesInLibrary = form.querySelector('.chkSkipEpisodesInLibrary').checked;
-            item.KeepUpTo = form.querySelector('.selectKeepUpTo').value;
+    context.querySelector('.btnCancel').addEventListener('click', function () {
+        closeDialog(false);
+    });
 
-            apiClient.updateLiveTvSeriesTimer(item);
+    context.querySelector('.btnCancelRecording').addEventListener('click', function () {
+        const apiClient = connectionManager.getApiClient(currentServerId);
+        deleteTimer(apiClient, currentItemId).then(function () {
+            closeDialog(true);
         });
+    });
 
-        e.preventDefault();
+    context.querySelector('form').addEventListener('submit', onSubmit);
+}
 
-        // Disable default form submission
-        return false;
-    }
+function reload(context, id) {
+    const apiClient = connectionManager.getApiClient(currentServerId);
 
-    function init(context) {
-        fillKeepUpTo(context);
+    loading.show();
+    if (typeof id === 'string') {
+        currentItemId = id;
 
-        context.querySelector('.btnCancel').addEventListener('click', function () {
-            closeDialog(false);
-        });
-
-        context.querySelector('.btnCancelRecording').addEventListener('click', function () {
-            var apiClient = connectionManager.getApiClient(currentServerId);
-            deleteTimer(apiClient, currentItemId).then(function () {
-                closeDialog(true);
-            });
+        apiClient.getLiveTvSeriesTimer(id).then(function (result) {
+            renderTimer(context, result);
+            loading.hide();
         });
+    } else if (id) {
+        currentItemId = id.Id;
 
-        context.querySelector('form').addEventListener('submit', onSubmit);
+        renderTimer(context, id);
+        loading.hide();
     }
+}
 
-    function reload(context, id) {
-        var apiClient = connectionManager.getApiClient(currentServerId);
-
-        loading.show();
-        if (typeof id === 'string') {
-            currentItemId = id;
+function fillKeepUpTo(context) {
+    let html = '';
 
-            apiClient.getLiveTvSeriesTimer(id).then(function (result) {
-                renderTimer(context, result, apiClient);
-                loading.hide();
-            });
-        } else if (id) {
-            currentItemId = id.Id;
+    for (let i = 0; i <= 50; i++) {
```

Comment:
```suggestion
    for (const i = 0; i <= 50; i++) {
```

IIRC @MrTimscampi has an eslint PR for this, but much better if we already save some bits to that PR